registry/client/repository.go

```diff
@@ -34,7 +34,7 @@ func NewRepository(ctx context.Context, name, baseURL string, transport http.Rou
 
 	client := &http.Client{
 		Transport: transport,
-		Timeout:   1 * time.Minute,
+		Timeout:   10 * time.Minute,
```

Comment:
Please add a `TODO` here explaining the trade off of selecting this value and why it is not ideal.